package.json

```diff
@@ -2,11 +2,11 @@
   "name": "epam-nodejs-practice",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon src/index.js"
+    "start": "npx nodemon"
```

Comment:
For dev server better to have something like `start:dev` script command. `start` is commonly used as default for running production build.